test/SlowTests/Sharding/Cluster/SubscriptionsWithReshardingTests.cs

```diff
@@ -1452,6 +1452,28 @@ await using (var subscription = store.Subscriptions.GetSubscriptionWorker<User>(
             return ((firstId, shardNumber1, bucketNumber1), (secondId, shardNumber2, bucketNumber2));
         }
 
+        internal static (string Id, int ShardNumber) GetIdOnDifferentShard(ShardingConfiguration conf, List<int> shards, string collection = "users")
```

Comment:
use `GetOneDocIdForEachShardAsync` method

Reply:
yes I saw it, as far as I understand `GetOneDocIdForEachShardAsync` returns you docs ids that _exist_ in storage (https://github.com/garayx/ravendb/blob/d234558abddd89371865faf88feb84be5ffbd612/test/Tests.Infrastructure/RavenTestBase.Sharding.cs#L404).

I am adding 2 docs to 2 different shard here: https://github.com/garayx/ravendb/blob/d234558abddd89371865faf88feb84be5ffbd612/test/SlowTests/Sharding/Subscriptions/ShardedSubscriptionSlowTests.cs#L1417

Then I need a new id that will be added to 3rd shard (before it exists in storage),
https://github.com/garayx/ravendb/blob/d234558abddd89371865faf88feb84be5ffbd612/test/SlowTests/Sharding/Subscriptions/ShardedSubscriptionSlowTests.cs#L1488

If I will use  `GetOneDocIdForEachShardAsync` here, I will get the existing ids (`id1` & `id2` in my test), and this is not what I need.

Reply:
so what about `GetRandomIdForShard` ?

Reply:
yes